fs/zbd_zenfs.cc

```diff
@@ -342,12 +340,10 @@ IOStatus ZonedBlockDevice::Open(bool readonly, bool exclusive) {
   return IOStatus::OK();
 }
 
-void ZonedBlockDevice::NotifyIOZoneFull() {
-  active_io_zones_--;
-  zone_resources_.notify_one();
-}
+void ZonedBlockDevice::NotifyIOZoneFull() { active_io_zones_--; }
```

Comment:
Why did you remove the notify here?

Reply:
It is as it says in the commit text. We don't use it.